src/javascript/static/pages/get_started.js

```diff
@@ -22,7 +24,19 @@ module.exports = {
         onUnload: () => { MenuSelector.clean(); },
     },
     BinaryOptionsForMT5: {
-        onLoad  : () => { MenuSelector.init(['what-are-binary-options', 'how-to-trade-binary', 'types-of-trades']); },
+        onLoad: () => {
+            let menu_sections = [
+                'what-are-binary-options',
+                'how-to-trade-binary',
+                'types-of-trades',
+            ];
+            const landing_company_shortcode = ClientBase.get('landing_company_shortcode');
+            if (landing_company_shortcode === 'malta' || landing_company_shortcode === 'maltainvest') {
+                getElementById('how-to-trade-binary').setVisibility(false);
+                menu_sections = menu_sections.filter(menu_item => menu_item !== 'how-to-trade-binary');
```

Comment:
is this better, or to push how-to-trade-binary into the array in else?

Reply:
I think this is better than pushing it in else since we remove one logical point we need to think of when reading the code. One thing that bothers me, is it not very obvious what it is trying to do, I wish JS have a `.remove` function that takes a value and remove it just like in Python, would make it more readable, but I still like this better because I can just think, 

`if landingcompany is malta or maltainvest, remove this section` instead of `if landingcompany is malta or maltainvest, remove this part otherwise add this part`